lstchain/io/io.py

```diff
@@ -77,6 +77,9 @@
 dl1_params_tel_mon_ped_key = "/dl1/event/telescope/monitoring/pedestal"
 dl1_params_tel_mon_cal_key = "/dl1/event/telescope/monitoring/calibration"
 dl1_params_tel_mon_flat_key = "/dl1/event/telescope/monitoring/flatfield"
+dl1_params_mon_tel_catB_ped_key = "/dl1/monitoring/telescope/CatB/pedestal"
+dl1_params_mon_tel_catB_cal_key = "/dl1/monitoring/telescope/CatB/calibration"
+dl1_params_mon_tel_catB_flat_key = "/dl1/monitoring/telescope/CatB/flatfield"
```

Comment:
these variables are being called differently in #1147 and #1174, without `_params`